src/java.desktop/share/classes/javax/swing/JLayeredPane.java

```diff
@@ -529,7 +529,7 @@ public int getComponentCountInLayer(int layer) {
             curLayer = getLayer(getComponent(i));
             if(curLayer == layer) {
                 layerCount++;
-            /// Short-circuit the counting when we have them all
+            /// Short circuit the counting when we have them all
```

Comment:
When used as a verb, it's usually spelt with the hyphen — *short-circuit* — as written here, see [“short-circuit” on Merriam Webster](https://www.merriam-webster.com/dictionary/short-circuit), [“short-circuit” on Collin's Dictionary](https://www.collinsdictionary.com/dictionary/english/short-circuit).

The [“short-circuit” entry on Wiktionary](https://en.wiktionary.org/wiki/short_circuit#Verb) lists the spelling without the hyphen, however, in notes it is mentioned, <q cite="https://en.wiktionary.org/wiki/short_circuit#Usage_notes">Senses 3 and 4 usually use the hyphenated spelling **short-circuit**.</q> I think the word is used in the sense 3.